guides/doc-Provisioning_Guide/topics/Resources.adoc

```diff
@@ -483,5 +483,10 @@ endif::[]
 # firewall-cmd --add-port=5900-5930/tcp
 # firewall-cmd --add-port=5900-5930/tcp --permanent
 ----
+ifeval::["{build}" == "foreman"]
```

Comment:
Does this ifeval need to be here, since the file itself also contains an ifeval?

Reply:
I need this ifeval to nest it with the ++ markup. Elseses it is a sentence that breaks the procedure. It's a bit like what Leo DiCaprio dealt with in that film Inception, I know, but with asciidoc, you take what is given to you. 

TL;DR: The + signs are the main object I need in that ifeval to frame the include.

Reply:
Asciidoc strikes again!